### PR TITLE
Symptom survey codebook and DUA user documentation

### PR DESCRIPTION
**Pull request by capnrefsmmat**
This adds documentation to the delphi-epidata docs for users of the survey microdata. This is based on the documentation already on the SFTP server, with some tweaking and updates, plus exported versions of all extant survey versions.

Demo version (link will change!): http://rosmarus.refsmmat.com/delphi-epidata/symptom-survey/

### CODE REVIEW

File: docs/symptom-survey/coding.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+order from left to right and top to bottom. When the respondent is allowed to
+select multiple responses, these are shown separated by commas, such as `2,4,6`.
+Questions left blank or with invalid responses are recorded in the CSV files as
+`NA`.
+
+Timestamps are provided in Pacific time (UTC-7). The following metadata columns
+describe each survey response:
+
+* `StartDatetime`: The time the respondent began the survey.
+* `EndDatetime`: The time the respondent's responses were recorded. This either
```
Comment (krivard) [inline docs/symptom-survey/coding.md:33]: No, that would be "RecordedDate" which is not included in these files.

"EndDatetime" is the last time of any respondent activity, which may be the same as the recorded time or four hours before the recorded time.

File: docs/symptom-survey/coding.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  coding](waves/Survey_of_COVID-Like_Illness_-_TODEPLOY-_US_Expansion_-_With_Translations.pdf)
+  (PDF)
+* [Survey text and
+  coding](waves/Survey_of_COVID-Like_Illness_-_TODEPLOY-_US_Expansion_-_With_Translations.docx)
+  (Word)
+
+### Summary of Changes
+
+* Consent text now mentions receiving "your language preference" from Facebook,
+  to allow Qualtrics to select the appropriate translation automatically.
```
Comment (krivard) [inline docs/symptom-survey/coding.md:109]:
```suggestion
* Consent text now mentions receiving "your language preference" from Facebook,
  to allow Qualtrics to select the appropriate translation automatically.
* Consent question now requires the respondent be located in the U.S.
```

File: docs/symptom-survey/survey-files.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+title: Response Files
+parent: COVID Symptom Survey
+nav_order: 1
+---
+
+# Response Files
+{: .no_toc}
+
+Users with access to the [COVID symptom survey](./index.md) aggregate data
```
Comment (krivard) [inline docs/symptom-survey/survey-files.md:10]:
```suggestion
Users with access to the [COVID symptom survey](./index.md) individual response data
```

File: docs/symptom-survey/survey-files.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+should have received SFTP credentials for a private server where the data are
+stored. This documentation describes the survey data available on that user.
+
+You should have signed a Data Use Agreement with Facebook and with CMU to gain
+access to these files. If you have not, aggregate data is available
```
Comment (krivard) [inline docs/symptom-survey/survey-files.md:15]:
```suggestion
You must a Data Use Agreement with Facebook and with CMU to gain
access to the individual survey responses. If you have not, aggregate data is available
```

File: docs/symptom-survey/survey-files.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+[through the COVIDcast API](../api/covidcast-signals/fb-survey.md).
+
+Important updates for data users, including corrections to data or updates on
+data processing delays, are posted as `OUTAGES.txt` in the SFTP server directory
+where the data is accessible.
```
Comment (krivard) [inline docs/symptom-survey/survey-files.md:20]:
```suggestion
where the data is hosted.
```

File: docs/symptom-survey/survey-files.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+1. TOC
+{:toc}
+
+## Naming Conventions
+
+All dates in filenames are of the form `YYYY_mm_dd`.
+
+Cumulative files:
+
+	cvid_responses_{from}_-_{to}.csv.gz
```
Comment (krivard) [inline docs/symptom-survey/survey-files.md:34]:
```suggestion
	`cvid_responses_{from}_-_{to}.csv.gz`
```

File: docs/symptom-survey/survey-files.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+Incremental files:
+
+	cvid_responses_{for}_recordedby_{recorded}.csv
```
Comment (krivard) [inline docs/symptom-survey/survey-files.md:38]:
```suggestion
	`cvid_responses_{for}_recordedby_{recorded}.csv`
```

File: docs/symptom-survey/survey-files.md (hunk 6 above)
Comment (capnrefsmmat) [reply docs/symptom-survey/survey-files.md:34]: Not needed in Markdown; indentation already turns this into a code block

**Pull request merged**